amplify-tools-gradle-plugin/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.amplifyframework.tools" />
```

Comment:
Please add the copy-right header that other xml files in the project are using.

----------------

If this module uses an `AndroidManifest.xml`, and does `import com.android.library`, it probably means that you can package your plugin as an `aar`. This would be good since our current `configuration/publishing.gradle` has been tested for publishing `aar`. Also, since all of the other modules are. This would mean you'd need to update the `POM_PACKACING` to `aar` as well of course.

------------


`tools` is pretty generic. We also have a "Plugin" concept in our client library. So, we need to be careful about the name we choose here. How about `com.amplifyframework.gradle`, or `com.amplifyframework.gradleplugin`

Reply:
iOS has a similar plugin for XCode, so we should maintain some naming consistency for our own sanity. @undefobj

Reply:
> iOS has a similar plugin for XCode, so we should maintain some naming consistency for our own sanity. @undefobj

yes please! :)

Reply:
Okay yes, I take back the name thing. :-D